test/e2e/authorization.go

```diff
@@ -198,39 +215,33 @@ var _ = g.Describe("Authorization [RBAC] [Zalando]", func() {
 
 		g.When("the resource is a namespaced resource", func() {
 			g.BeforeEach(func() {
-				tc.data.resources = []string{
-					"pods",
-					"apps/deployments",
-					"apps/statefulsets",
-					"apps/deployments/scale",
-					"apps/statefulsets/scale",
-					"services",
-					"persistentvolumes",
-					"persistentvolumeclaims",
-					"configmaps",
-				}
-				tc.data.verbs = []string{"create", "update", "delete", "patch"}
-			})
-			// These should be covered by the admission-controller tests.
-			// They're written here for completeness.
-			g.It("should deny write access in kube-system and visibility namespaces", func() {})
-			g.It("should allow write access in namespaces other than kube-system and visibility", func() {})
+				tc.data.resources = namespacedResources
+				tc.data.verbs = writeOperations
+			})
+			// These should be covered by the admission-controller tests. They will
+			// be skipped here. Later when we cover everything with RBAC, we can run them again.
+			g.It("should deny write access in kube-system and visibility namespaces", func() {
+				tc.data.namespaces = []string{"kube-system", "visibility"}
+				tc.run(context.TODO(), cs, false)
+				gomega.Expect(tc.output.passed).To(gomega.BeTrue(), tc.output.String())
+			})
+			g.It("should allow write access in namespaces other than kube-system and visibility", func() {
+				tc.data.namespaces = []string{"", "teapot"}
```

Comment:
Given that the test is about namespaced resources, how does it play together with `""` (global namespace) here? Does the test do the right thing?

Reply:
Looking at the authv1 library definition, it says that `""` for a namespaced resources means "all namespaces". This now has me concerned. I think maybe it's best to replace `""` with `"default"`. What do you think?

Good catch!

Reply:
I fixed this `re/""/"default"/` in the next commit.